docs/src/convert-to-onda.md

```diff
@@ -190,17 +191,96 @@ new_plan = plan_edf_to_onda_samples_groups(new_plan)
 ### Execute the plan
 
 Once the plan has been reviewed and deemed satisfactory, execute the plan to generate `Onda.Samples` and an "executed plan" record.
-This is accomplished with the [`edf_to_onda_samples`](@ref) function, which takes an `EDF.File` and a plan as input, and returns a vector of `Onda.Samples` and the plan as executed.
-The executed plan may differ from the input plan.
-Most notably, if any errors were encountered during execution, they will be caught and the error and stacktrace will be stored as strings in the `error` field.
-It is important to review the executed plan a final time to ensure everything was converted as expected and no unexpected errors were encountered.
+This is accomplished with the [`edf_to_onda_samples`](@ref) function, which takes an `EDF.File` and a plan as input, and returns a `Vector{ConvertedSamples}`.
```

Comment:
Just a thought. Since the function is `edf_to_onda_samples` it may make more sense for the struct to be named `ConvertedOndaSamples`

Reply:
I think the name (which was @ericphanson suggestion originally) comes from the fact that it wraps a `Samples` object.  could make the same argument for the function name too though but I'd rather keep it more concise as long as it's clear (and I think it is...)